common/src/deployment.rs

```diff
@@ -56,7 +56,7 @@ pub const DEPLOYER_END_MSG_BUILD_ERR: &str = "Service build encountered an error
 pub const DEPLOYER_END_MSG_CRASHED: &str = "Service encountered an error and crashed";
 pub const DEPLOYER_END_MSG_STOPPED: &str = "Service was stopped by the user"; // don't include this in end messages so that logs are not stopped too early
 pub const DEPLOYER_END_MSG_COMPLETED: &str = "Service finished running all on its own";
-pub const DEPLOYER_RUNTIME_START_RESPONSE: &str = "Runtime started successully";
+pub const DEPLOYER_RUNTIME_START_RESPONSE: &str = "Runtime started successfully";
```

Comment:
Sadly, fixing this one is a breaking change. Will come back when I know what to do about it.

Reply:
One of those horror stories where someone relies on a string match of an error or other kind of log? :O

Reply:
Either way you should have edit rights, feel fee to revert this change and keep it for another time.

Reply:
I might do that and attempt at making this more robust. Context about this design: When we reworked our logging system, we dumbed down some parts, for better or for worse.